Dockerfile

```diff
@@ -20,10 +20,10 @@ ENV KUBERMATIC_CHARTS_DIRECTORY=/opt/charts/
 # To support a wider range of Kubernetes userclusters, we ship multiple
 # kubectl binaries and deduce which one to use based on the version skew
 # policy.
-ADD https://storage.googleapis.com/kubernetes-release/release/v1.26.4/bin/linux/amd64/kubectl /usr/local/bin/kubectl-1.26
-ADD https://storage.googleapis.com/kubernetes-release/release/v1.24.13/bin/linux/amd64/kubectl /usr/local/bin/kubectl-1.24
+ADD https://storage.googleapis.com/kubernetes-release/release/v1.26.6/bin/linux/amd64/kubectl /usr/local/bin/kubectl-1.26
+ADD https://storage.googleapis.com/kubernetes-release/release/v1.24.15/bin/linux/amd64/kubectl /usr/local/bin/kubectl-1.24
 
-RUN wget -O- https://get.helm.sh/helm-v3.10.1-linux-amd64.tar.gz | tar xzOf - linux-amd64/helm > /usr/local/bin/helm
+RUN wget -O- https://get.helm.sh/helm-v3.11.3-linux-amd64.tar.gz | tar xzOf - linux-amd64/helm > /usr/local/bin/helm
```

Comment:
Minor q, can we also bump the Helm version in `go.mod` so we align both? The Go dependency is on v3.11.1, so shouldn't be a major change.

Reply:
Not bumping go.mod in favour of: https://github.com/kubermatic/kubermatic/pull/12442